### PR TITLE
Loading screen text issue fixed

### LINKED ISSUE #12580: [UI] Text under meshery logo in loading screen stays black.
### Current Behavior
In the loading screen the color of the meshery text under the logo is always black.

### Desired Behavior

The logo should be changed according to the user preference.

### Screenshots/Logs
Screenshot from 2024-11-05 22-08-47

### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**

- This PR fixes #12580 

Fixed the loading screen text issue under the meshery logo:
Screenshot from 2024-11-07 20-00-45

(updated Screenshot)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Pratik-050** [comment]
It gets the job done for now @leecalcote!
Please take a look @sudhanshutech @SAHU-01, Thanks!

File: ui/components/LoadingComponents/LoadingComponentServer.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ import React from 'react';
 // import AnimatedMeshery from './Animations/AnimatedMesheryCSS';
 import PropTypes from 'prop-types';
 import uiConfig from '../../ui.config';
+import { useTheme } from '@material-ui/core';
```
Comment (leecalcote) [inline ui/components/LoadingComponents/LoadingComponentServer.js:5]: Use Sistent.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/LoadingComponents/LoadingComponentServer.js [hunk 2]
```diff
@@ -21,7 +23,10 @@ function LoadingScreen(props) {
...
       >
         <div>
-          <AnimatedLogoDark style={{ height: '100px', margin: '4px 0px 8px' }} />
+          <AnimatedLogoDark
+            style={{ height: '100px', margin: '4px 0px 8px' }}
+            textFill={theme.palette.type === 'dark' ? '#fff' : ''}
+          />
           <h1
             style={{
               fontFamily: 'sans-serif',
```
Comment (leecalcote) [inline ui/components/LoadingComponents/LoadingComponentServer.js:32]: Change to Sistent's theme text. Use Qanelas Soft here, please.

Reduce to an h2 or h3 - whatever is the size used when loading the final "Meshery Extensions" loading screen.

**Comment by vishalvivekm** [comment]
@Pratik-050 Thank you for your contributions! Let's discuss this during Wednesday's Meshery Development call at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/).

File: ui/components/LoadingComponents/LoadingComponentServer.js (hunk 2 above)
Comment (leecalcote) [reply ui/components/LoadingComponents/LoadingComponentServer.js]: Confirm that these settings are the same, @Pratik-050

Comment (leecalcote) [reply ui/components/LoadingComponents/LoadingComponentServer.js]: That’s not what I’m asking. There need to be changes in order for these two different loading screens to be the same. Confirm that there are no differences between the two, and if there are, make changes to make to align them.

Comment (Pratik-050) [reply ui/components/LoadingComponents/LoadingComponentServer.js]: No sir everything is working fine, I'm just conditionally passing the color according to the theme and changed the div to the sistent typography component. That's all thanks!

Comment (leecalcote) [reply ui/components/LoadingComponents/LoadingComponentServer.js]: @Pratik-050, again, confirm the answers to my question regarding font family and font size.

Comment (Pratik-050) [reply ui/components/LoadingComponents/LoadingComponentServer.js]: Hello sir, @leecalcote the font style is changed as said, and there is no field for font-size the token received from the sistent, the other component you've mentioned is using MUI, so I've made the font 16px that is h3 from mui library. Other than that we are OK! Thanks.

**Pull request merged**